whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -116,18 +116,18 @@
       "EncLevelType": {
         "1": "marc:FullLevelMaterialNotExamined",
         "2": "marc:LessThanFullLevelMaterialNotExamined",
-        "3": "marc:AbbreviatedLevel",
-        "4": "marc:CoreLevel",
+        "4": "marc:AbbreviatedLevel",
         "5": "marc:PartialPreliminaryLevel",
         "7": "marc:MinimalLevel",
         "8": "marc:PrepublicationLevel",
         " ": "marc:FullLevel",
         "e": "marc:SystemIdentifiedMarcErrorInABatchloadedRecordLocal",
-        "i": "marc:FullLevelInputByOclcParticipantsLocal",
-        "j": "marc:DeletedRecordLocal",
-        "k": "marc:LessThanFullLevelInputByOclcParticipantsLocal",
-        "l": "marc:FullLevelInputAddedFromABatchProcessLocal",
-        "m": "marc:LessThanFullLevelInputAddedFromABatchProcessLocal"
+        "i": "marc:AbbreviatedLevel",
+        "j": "marc:AbbreviatedLevel",
+        "k": "marc:AbbreviatedLevel",
+        "l": "marc:AbbreviatedLevel",
+        "m": "marc:AbbreviatedLevel",
+        "3": "marc:AbbreviatedLevel"
```

Comment:
I want to discuss this part: To be able to make marc:AbbreviatedLevel revert to 3, I moved the declaration of token 3 to the end of list. We would need a more fullproof way of ensuring that marc:AbbreviatedLevel always reverts to 3. Or is this enough?